packages/create-react-app/createReactApp.js

```diff
@@ -49,7 +49,7 @@ const url = require('url');
 const hyperquest = require('hyperquest');
 const envinfo = require('envinfo');
 const os = require('os');
-const monorepo = require('react-dev-utils/monorepo');
+const getMonorepo = require('react-dev-utils/workspaceUtils').getMonorepo;
```

Comment:
How about `findMonorepo`? Sorry for bikeshedding on this.

Users who don't use monorepos might be confused after ejecting because how would you "get" a monorepo that doesn't exist.